scrapyd_k8s.sample-k8s.conf

```diff
@@ -57,3 +57,16 @@ requests_cpu = 80m
 requests_memory = 0.12G
 limits_cpu = 0.5
 limits_memory = 0.2G
+
+;[joblogs]
+;# Choose storage provider
+;storage_provider = s3
+;container_name   = <container_name>
+
+;# Choose number of unique logs, but at least 2
+;num_lines_to_check = 2
+
+;[joblogs.s3]
```

Comment:
Let's name this `joblogs.storage.s3`, to avoid any future collison when there may be a need for more sections.